src/fabric-video/index.scss

```diff
@@ -2,6 +2,7 @@
 
 .creative--fabric-video {
     margin: auto;
+    max-width: 1920px;
```

Comment:
Would we benefit from storing this in a variable somewhere, e.g. `ultimate-super-wide-width-width`? :)